iree/compiler/Dialect/VM/Target/C/test/constant_ops.mlir

```diff
@@ -14,8 +14,8 @@ vm.module @constant_ops {
   // CHECK-NEXT: iree_vm_buffer_t rodata_buffers[2];
   // CHECK-NEXT: };
 
-  vm.rodata private @buffer_1 dense<[1, 2, 3]> : tensor<3xi8>
-  vm.rodata private @buffer_2 dense<[1, 2, 3]> : tensor<3xi32>
+  vm.rodata public @buffer_1 dense<[1, 2, 3]> : tensor<3xi8>
```

Comment:
rodata should in general be private - there's no way to reference them outside of the module (can't export rodata) and it's better to have symbol DCE be able to drop them

Reply:
Sure but I don't want DCE to kick in here. I shamelessly copied your formulation to a comment :P